frontend/src/components/Card/Card.test.tsx

```diff
@@ -1,24 +1,50 @@
-import { render, screen } from "@testing-library/react";
-import { describe, expect, it } from "vitest";
+import { screen } from "@testing-library/react";
 import { axe } from "vitest-axe";
 
+import { renderWithRouter } from "@/lib/test/renderWithRouter";
+
 import { Card } from "./Card";
 
 describe("Given a user is on the home page", () => {
   describe("When the Card component is rendered", () => {
     it("Then the user is seeing the correct product", () => {
```

Comment:
💅 Just a little bit more specific comments that someone what the purpose of card is and what we should see each time card would be rendered 

```suggestion
describe("Given a user is on the home page", () => {
  describe("When the Card component is rendered with a product", () => {
    it("Then the user should see the product's details", () => {
```

Reply:
I removed the test for the product price, title and supermarket too, as these are also displayed in the carddetails component